frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -179,10 +179,10 @@ export const apiCreateQuestion = question => {
       .setResultsMetadata(resultsMetadata)
       .reduxCreate(dispatch);
 
-    // remove the databases in the store that are used to populate the QB databases list.
-    // This is done when saving a Card because the newly saved card will be eligible for use as a source query
-    // so we want the databases list to be re-fetched next time we hit "New Question" so it shows up
-    dispatch(setRequestUnloaded(["entities", "databases"]));
+    const databases = Databases.selectors.getList(getState());
+    if (databases && !databases.some(d => d.is_saved_questions)) {
```

Comment:
It isn't possible to write a reducer that updates not individual entities but list results, so let's do what the deleted line was supposed to do but conditionally to avoid extra requests.